service/error.go

```diff
@@ -44,35 +44,3 @@ func (e *Error) WithMeta(meta interface{}) *Error {
 	e.Meta = meta
 	return e
 }
-
-// TODO: Deprecate below Errors struct
```

Comment:
you answered my question - "why is there code with a Deprecate tag"  :) all clean now